lib/OpenQA/WebAPI/Controller/Step.pm

```diff
@@ -154,7 +154,7 @@ sub edit ($self) {
                     width => int $match->{w},
                     ypos => int $match->{y},
                     height => int $match->{h},
-                    type => 'match',
+		    type => 'match',
```

Comment:
This change looks unintended. Likely tidy will fix it for you (which you'll have to run anyways).